.lando.yml

```diff
@@ -139,7 +139,7 @@ services:
         DRUPAL_REVALIDATE_SECRET: revalidate_secret_not_secure_used_only_locally
         DRUPAL_CLIENT_SECRET: drupal_client_secret_not_secure_used_only_locally
         DRUPAL_CLIENT_ID: drupal-client-id
-        NEXT_PUBLIC_DRUPAL_BASE_URL: https://next-drupal-starterkit.lndo.site
```

Comment:
using https with this version of `next dev` caused this issue:

```
 TypeError: fetch failed
    at Object.fetch (node:internal/deps/undici/undici:11457:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async exports.DrupalClient.fetch (/app/next/node_modules/next-drupal/dist/index.js:1:9978)
    at async exports.DrupalClient.getIndex (/app/next/node_modules/next-drupal/dist/index.js:1:17631)
    at async exports.DrupalClient.getEntryForResourceType (/app/next/node_modules/next-drupal/dist/index.js:1:18010)
    at async exports.DrupalClient.getResourceCollection (/app/next/node_modules/next-drupal/dist/index.js:1:15013)
    at async exports.DrupalClient.getResourceCollectionFromContext (/app/next/node_modules/next-drupal/dist/index.js:1:15458)
    at async getStaticProps (webpack-internal:///./pages/index.tsx:106:24) {
  cause: Error: unable to verify the first certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:959:8)
      at ssl.onhandshakedone (node:_tls_wrap:743:12) {
    code: 'UNABLE_TO_VERIFY_LEAF_SIGNATURE'
  }
}
``` 

So, since this is only the local environment, we are switching to use the http version that lando also provides.

Reply:
In our .env.local we have this setting which may or may not help you here:
```
# Needed for images to work locally
NODE_TLS_REJECT_UNAUTHORIZED=0 

```

Reply:
hi @badrange ! 

thanks, that's the other option that we are considering. It does output a warning each time which is not great though. :/ 

I think the issue here is that we are doing everything inside lando, and it gets a bit complicated because lando has its own way to deal with https, which sounds like  it interferes with the newer and faster version of npm run dev

We'll try to fix this in the next days.

Reply:
Cheers! Maybe you can find a way to suppress that warning; since it is only for local development it doesn't really matter. Seems a bit of a hassle to set up proper certificate trusting of lando's generated certificates for nodejs. But if you can manage to keep everything http only then sure that's probably even better.

Reply:
There's a new version of next.js (13.15.4) and with that one the issue with https seems to have been fixed.

Reply:
It looks like in the newer version the issue is indeed fixed.

Reply:
Good to know!